### DISCUSSION TITLE
How to reset the controller?

### DISCUSSION THREAD

**Discussion by tomcatxx** [Dlloydev/QuickPID#71] [Q&A]
Hi how can I reset die the PID Controller and its internal values?
For examle to start fresh without old values after switched back from manual to automatic.

**Comment by Dlloydev** [comment]
I've just published a new release: QuickPID 3.1.8
* Added  `Reset()` function that clears the `pTerm`, `iTerm`, `dTerm` and `outputSum` values.
* ReadME updated

Example use: 
`QuickPID myPID(&Input, &Output, &Setpoint);  // simplest constructor
`

Then in setup or loop:
`myPID.Reset();`

**Reply by tomcatxx** [reply]
Thank you!
